re-worded event from glossary (#40935)
kubernetes/website: content/en/docs/reference/glossary/event.md

date: 2022-01-16
full_link: /docs/reference/kubernetes-api/cluster-resources/event-v1/
short_description: >

>    Refers to an object that describes a notable occurrence or change of state in the system.

Reviewer: WDYT?
```suggestion
Events are Kubernetes objects that describe some state change in the system.
```

>    Refers to an object that describes a notable occurrence or change of state in the system. 

Reviewer 2: ```suggestion
   Refers to an object that describes a notable occurrence or change of state in the system.
```
Refers what? Also, I'm not sure why we're adding an extra space. Can you explain if this was a typo?

Author: Yes, I am have added a space while typing mb.

Author: > Refers what? Also, I'm not sure why we're adding an extra space. Can you explain if this was a typo?

"refers to an object" is not the correct way to put it?

aka: 
tags:
- core-object
- fundamental
---

> An Event refers to an object that describes a notable occurrence in the system. 

Reviewer: I think the suggested one is unnecessarily verbose.The event is a Kubernetes object that describes state change/notable occurrences in the system.
